README.md

```diff
@@ -1,11 +1,54 @@
+
 # Truffle Suite Website - https://www.truffleframework.com
 
-Our website is built with [Metalsmith](http://www.metalsmith.io/), using Handlebars templates and a combination of HTML and markdown for page content. 
+  
+
+Our website is built with [Metalsmith](http://www.metalsmith.io/), using Handlebars templates and a combination of HTML and markdown for page content.
+
+  
 
 Check out [the contribution guide](https://github.com/trufflesuite/trufflesuite.com/blob/master/CONTRIBUTING.md) to get a lay of the land.
 
+  
+
 ## Contributing
 
+  
+
 We welcome all contributions, typo corrections, and general feedback. See our [full contribution guide](https://github.com/trufflesuite/trufflesuite.com/blob/master/CONTRIBUTING.md) for details on site architecture and how you could help.
 
-If you find an issue while browsing, most pages have a GitHub link to the directory containing their source files. Look out for the "See a way to make this page better?" links.
\ No newline at end of file
+  
+
+If you find an issue while browsing, most pages have a GitHub link to the directory containing their source files. Look out for the "See a way to make this page better?" links.
+
+  
+
+## Getting Started
+Download zip file or git clone repo:
+```
+git clone https://github.com/trufflesuite/trufflesuite.com.git
+```
+Navigate into the directory in terminal:
+```
+cd trufflesuite.com/
+npm install
+```
+
+## Run dev build
+In terminal run: 
+```
+npm run dev
+```
+Navigate to **localhost:9000**/port specified in your terminal
```

Comment:
Can you please turn this into a hyperlink? It'll be easier that way!

Reply:
Done!